canary/webrtc-c/src/CloudwatchMonitoring.cpp

```diff
@@ -166,6 +166,16 @@ VOID CloudwatchMonitoring::pushSignalingRoundtripLatency(UINT64 delay, StandardU
     this->push(datum);
 }
 
+VOID CloudwatchMonitoring::pushStartUpLatency(UINT64 latency, StandardUnit unit)
+{
+    MetricDatum datum;
+
+    datum.SetMetricName("StartUpLatency");
```

Comment:
Should this be named as "TimeToFirstFrame"? or something similar? Startup latency sounds a little vague